ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1076,66 +1076,74 @@ def test024_archive_sub_order(self):
         results = self.order_page.result_table(element='general:table_child')[0].text
         self.assertIn(suborders_data[0]['Analysis No.'].replace("'", ""), results.replace("'", ""))
 
-    def test027_update_material_type(self):
+    @parameterized.expand(['testPlans', 'testUnit'])
+    def test027_update_material_type(self, case):
         """
-        Apply this on suborder number 5 for example:-
         -When user update the materiel type from table view once I delete it message will appear
-        (All analysis created with this order and test plan will be deleted )
-        -Once you press on OK button, the material type & article & test pan will delete
-        -You can update it by choose another one and choose corresponding article & test plan
-        LIMS-4264
-        """
-        self.info('get random order')
-        orders, _ = self.orders_api.get_all_orders(limit=50)
-        order = random.choice(orders['orders'])
-        suborders, _ = self.orders_api.get_suborder_by_order_id(order['id'])
-        suborder = suborders['orders'][0]
-        suborder_update_index = len(suborders['orders']) - 1
+        (All analysis created with this order and test plan/ test unit will be deleted )
+        -Once you press on OK button, the material type & article & test pan/ test unit will delete
+        -You can update it by choose another one and choose corresponding article & test plan/ test unit
 
+        LIMS-4264 ( order with test plan )
+
+        LIMS-4267 (order with test unit )
+        """
+        self.info('create new order')
+        api, order_payload = self.orders_api.create_new_order()
+        self.assertEqual(api['status'], 1, api)
+        self.info('new order created with order data {}'.format(order_payload))
```

Comment:
remove it

Reply:
line 1094 only right ? as you selected multiple lines

Reply:
yes only 1094

Reply:
done